qa/tasks/cephfs/test_failover.py

```diff
@@ -706,6 +708,78 @@ def test_rank_stopped(self):
         status = self._confirm_single_replay()
         self.assertTrue(standby_count, len(list(status.get_standbys())))
 
+    def _teardown(self, open_proc):
```

Comment:
Any other callers? Why have a separate function for this?

Reply:
Since there's no way to break out of `safe_while()`, we have to return form within the loop. And we can't exit as it is because it might affect default CephFSTestCase.tearDown() code. And it was getting too cluttered there since we already had too many levels of indentation. So I decided to move that code away to a helper.

Reply:
Removed the teardown method.